nva-datamodel-java/src/main/java/no/unit/nva/model/associatedartifacts/file/File.java

```diff
@@ -128,6 +134,7 @@ protected File(
         this.embargoDate = embargoDate;
         this.rightsRetentionStrategy = assignDefaultStrategyIfNull(rightsRetentionStrategy);
         this.legalNote = legalNote;
+        this.inserted = inserted;
```

Comment:
not a fan of this name myself.

Reply:
FileProvenance, but is it really a better name, <insert my name> :D

Reply:
Changed to uploadDetails